### PR TITLE
feat: Add testcafe tests

### PR DESCRIPTION
**Pull request by CozyKourai**
- Testcafé tests
- Travis config
- New runner

### CODE REVIEW

File: testcafe/config.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "loginUrl": "https://testcafe.cozy.works",
+  "photosUrl": "https://testcafe-photos.cozy.works/",
+  "password": "t€Stc@fé"
+}
```
Comment (y-lohse) [inline testcafe/config.json]: I think we can delete this file now, right?

File: testcafe/tests/pages/photos-model.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    this.barPhotoBtnDelete = this.barPhoto.find('button').nth(2) //DELETE
+    this.modalDelete = Selector('[class*="c-modal"]').find('div')
+    this.modalDeleteBtnDelete = this.modalDelete.find('button').nth(2) //REMOVE
+    this.allPhotos = Selector('[class*="pho-photo"]').find(
+      '[class*="pho-photo-item"]'
+    )
+  }
+
+  async uploadPhotos(files) {
+    var numOfFiles = files.length
```
Comment (y-lohse) [inline testcafe/tests/pages/photos-model.js]:
```suggestion
    const numOfFiles = files.length
```

File: testcafe/config.json (hunk 1 above)
Comment (Crash--) [inline testcafe/config.json]: @CozyKourai a supprimer stp

File: testcafe/tests/pages/photos-model.js (hunk 2 above)
Comment (Crash--) [reply testcafe/tests/pages/photos-model.js]: @CozyKourai a corriger

**Comment by CozyKourai** [comment]
[Tests E2E - Amélioration des tests existants et retours de PR](https://trello.com/c/lBMoXXux/1551-tests-e2e-am%C3%A9lioration-des-tests-existants-et-retours-de-pr)

**Pull request merged**